src/client/theme-default/components/VPNavBar.vue

```diff
@@ -52,8 +52,8 @@ const { hasSidebar } = useSidebar()
   position: relative;
   border-bottom: 1px solid var(--vp-c-divider-light);
   padding: 0 8px 0 24px;
-  height: var(--vp-nav-height-mobile);
-  transition: border-color 0.5s, background-color 0.5s;
+  min-height: calc(var(--vp-nav-height-mobile) + 1px);
```

Comment:
Two things here:

- We use a min-height rather than a fixed height for the NavBar. (This will be changed for the large layout a bit lower.)
- We need to account for the NavBar's border. The NavBar contains many elements which have a `height: var(--vp-nav-height-mobile)`. That means that if we use a fixed height, elements inside the NavBar will be painted on top of the border, and may look slightly misaligned. They will also obscure the border if they ever get an opaque or semi-opaque background. I think it's better to make the total height of the NavBar be the specified height plus one pixel for the border (or to make the height of NavBar content use `calc(var(--vp-nav-height-mobile) - 1px)` instead, but that's not ideal for optical alignment).